build.gradle

```diff
@@ -3,10 +3,26 @@ description = "Apereo uPortal $version"
 // Build Scans provide detailed information about many aspects of your
 // build right in your browser.  (https://scans.gradle.com/plugin)
 apply plugin: 'com.gradle.build-scan'
+apply plugin: 'java'
 
 // Adds support for Node.js scripts
 apply plugin: 'com.moowork.node'
 
+def props = new Properties()
+
+task copySampleProperties(type: Copy) {
+ printf "Configuring copying the build.properties.sample into build.properties"
+ from '.'
+    into '.'
+    include 'build.properties.sample'
+    rename { String fileName ->
+    fileName.replace("build.properties.sample", "build.properties")
+  }
+  doLast {
+	  file("build.properties").withInputStream { props.load(it) }
+  }
+}
+
```

Comment:
@bjagg @stalele CI either needs to be able to lazy load properties or to copy the file over.
Currently the copy approach is being taken, and gradle gives a cross platform approach to handle copying.